From c82bb00e8ccd53d35b66dc86418fdc5af2848efb Mon Sep 17 00:00:00 2001
From: Paul Harrison <pharrison@mythtv.org>
Date: Wed, 19 Mar 2014 17:05:30 +0000
Subject: [PATCH 13/16] MusicFileScanner: initialise some member variables in
 the ctor

This is really a false positive because they are all set to 0 before use in
SearchDirs() anyway. Fixes Coverity ID #1164504
---
 mythtv/libs/libmythmetadata/musicfilescanner.cpp |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/mythtv/libs/libmythmetadata/musicfilescanner.cpp b/mythtv/libs/libmythmetadata/musicfilescanner.cpp
index 4b20347..f036e8b 100644
--- a/mythtv/libs/libmythmetadata/musicfilescanner.cpp
+++ b/mythtv/libs/libmythmetadata/musicfilescanner.cpp
@@ -13,7 +13,10 @@
 #include <metaio.h>
 #include <musicfilescanner.h>
 
-MusicFileScanner::MusicFileScanner()
+MusicFileScanner::MusicFileScanner():
+    m_tracksTotal(0), m_tracksUnchanged(0), m_tracksAdded (0), m_tracksRemoved(0),
+    m_tracksUpdated(0), m_coverartTotal(0), m_coverartUnchanged(0), m_coverartAdded(0),
+    m_coverartRemoved(0), m_coverartUpdated(0)
 {
     MSqlQuery query(MSqlQuery::InitCon());
 
-- 
1.7.10.2

